### PR TITLE
Remove or ignore unused imports (F401)

### PR DESCRIPTION
**Pull request by winniehell**
## Description of the issue/feature this PR addresses

Enables flake8 check for unused imports (F401) in Travis CI.

### CODE REVIEW

File: travis_ci_flake8.cfg [hunk 1]
```diff
@@ -133,3 +129,15 @@ ignore =
     W601,
     # W605: invalid escape sequence '\d'
     W605,
+per-file-ignores =
```
Comment (winniehell) [inline travis_ci_flake8.cfg:132]: please double check if all of these are intentional

File: travis_ci_flake8.cfg [hunk 2]
```diff
@@ -85,16 +85,12 @@ ignore =
     E701,
     # E703: statement ends with a semicolon
     E703,
-    # E712: comparison to False should be 'if cond is False:' or 'if not cond:'
-    E712,
```
Comment (winniehell) [inline travis_ci_flake8.cfg:89]: follow-up of https://github.com/senaite/senaite.core/pull/2079

File: src/bika/lims/browser/worksheet/tools.py [hunk 3]
```diff
@@ -38,7 +38,7 @@ def checkUserAccess(worksheet, request, redirect=True):
     """
     # Deny access to foreign analysts
     allowed = worksheet.checkUserAccess()
-    if allowed == False and redirect == True:
```
Comment (winniehell) [inline src/bika/lims/browser/worksheet/tools.py:41]: I forgot this file in https://github.com/senaite/senaite.core/pull/2079

the change seemed small enough to be included here

File: src/bika/lims/browser/analysisrequest/__init__.py [hunk 4]
```diff
@@ -34,9 +34,9 @@
 # this comment twice.
 from .view import AnalysisRequestViewView
 
-from .add2 import AnalysisRequestAddView  # noqa: F401
-from .add2 import AnalysisRequestManageView  # noqa: F401
-from .add2 import ajaxAnalysisRequestAddView  # noqa: F401
```
Comment (ramonski) [inline src/bika/lims/browser/analysisrequest/__init__.py:39]: Why is it not necessary to have these lines? Are they no longer reported as "unused import"?

File: travis_ci_flake8.cfg [hunk 5]
```diff
@@ -133,3 +129,15 @@ ignore =
...
+    src/bika/lims/browser/worksheet/views/__init__.py:F401,
+    src/bika/lims/browser/analyses/__init__.py:F401,
+    src/bika/lims/browser/client/__init__.py:F401,
+    src/bika/lims/browser/widgets/__init__.py:F401,
+    src/bika/lims/browser/analysisrequest/analysisrequests.py:F401,
+    src/bika/lims/browser/analysisrequest/__init__.py:F401,
+    src/bika/lims/subscribers/__init__.py:F401,
+    src/bika/lims/interfaces/__init__.py:F401,
+    src/senaite/core/exportimport/instruments/__init__.py:F401,
+    src/senaite/core/behaviors/__init__.py:F401,
```
Comment (ramonski) [inline travis_ci_flake8.cfg:143]: ah, got it, here you ignore them per file.
However, probably the local editor will still complain, but ok for now.

**Review by ramonski — APPROVED** [review]
Thanks for your clean-up efforts @winniehell!

File: travis_ci_flake8.cfg (hunk 5 above)
Comment (winniehell) [reply travis_ci_flake8.cfg:143]: >  However, probably the local editor will still complain, but ok for now.

good point :+1: maybe we can rename the config to `.flake8` after cleaning up and hopefully the editor(s) will recognize it?

**Pull request merged**